### PR TITLE
[Did you know] Tip to redirect output ex-command

### PR DESCRIPTION
**Pull request by Wansmer**

### CODE REVIEW

File: contents/2023/Jan/23/5-did-you-know.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+```
+:put =execute('messages')
+```
+
+If you don't want to use no current buffer, and you want to specify filename for redirect output – use built-in Ex-command `:redir`.
+In additional, for redirect shell-command in current buffer, use `!!` in normal mode or `:.!` in cmd-line mode:
+
+```
+:.!tree
```
Comment (bew) [inline contents/2023/Jan/23/5-did-you-know.md:14]: hmm note that `:.!something` (and thus `!!` in normal mode) will replace the current line in the buffer, that should be mentionned.
There is also `:r !something` to put output of `something` after current line (`:r` is for `:read`)

File: contents/2023/Jan/23/5-did-you-know.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
 # [Did you know?](#tips) {#tips}
 
+We [continue](https://this-week-in-neovim.org/2023/Jan/16#tips) to learn `=` operator in command-line mode. Sometimes you need to redirect output of **Ex** command to current buffer.
+You can do it with `:put` command, `execute()` and `=` operator:
+
+```
+:put =execute('messages')
+```
+
+If you don't want to use no current buffer, and you want to specify filename for redirect output – use built-in Ex-command `:redir`.
```
Comment (bew) [inline contents/2023/Jan/23/5-did-you-know.md:10]: Could you show an example of using `:redir` ?

Comment (bew) [inline contents/2023/Jan/23/5-did-you-know.md:10]: Also, `don't want to use no...` is not really good english if I'm not mistaken, I think this is better:
```suggestion
If you don't want to use the current buffer, and you want to specify filename for redirect output – use built-in Ex-command `:redir`.
```

Comment (Wansmer) [reply contents/2023/Jan/23/5-did-you-know.md:10]: Yep, you are right, thanks for suggestion

Comment (Wansmer) [reply contents/2023/Jan/23/5-did-you-know.md]: Added with [e5492b3](https://github.com/phaazon/this-week-in-neovim-contents/pull/162/commits/e5492b3aaec07634aebe9643b7d01ab57d7c0903)

File: contents/2023/Jan/23/5-did-you-know.md (hunk 1 above)
Comment (Wansmer) [reply contents/2023/Jan/23/5-did-you-know.md:21]: Thanks, good note. Added it in text.

File: contents/2023/Jan/23/5-did-you-know.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
 # [Did you know?](#tips) {#tips}
 
+We [continue](https://this-week-in-neovim.org/2023/Jan/16#tips) to learn `=` operator in command-line mode. Sometimes you need to redirect output of **Ex** command to current buffer.
+You can do it with `:put` command, `execute()` and `=` operator:
+
+```
+:put =execute('messages')
+```
+
+If you don't want to add to the current buffer, and you can also redirect commands output to a file, register or variable using built-in Ex-command `:redir`:
```
Comment (bew) [inline contents/2023/Jan/23/5-did-you-know.md:10]: Made a typo in my suggestion earlier 👀
```suggestion
If you don't want to add to the current buffer, you can also redirect commands output to a file, register or variable using built-in Ex-command `:redir`:
```

**Review by phaazon — APPROVED** [review]
Thank you! Please always think about squashing your commits. I’m doing it for you for merging.

**Pull request merged**